tests/CecilTests/CompilerTestBase.cs

```diff
@@ -15,19 +14,27 @@ public partial class CompilerTestBase
         static CecilTypeSystem CreateCecilTypeSystem()
         {
             var self = typeof(CompilerTestBase).Assembly.GetModules()[0].FullyQualifiedName;
-            #if USE_NETSTANDARD_BUILD
+#if USE_NETSTANDARD_BUILD
             var selfDir = Path.GetDirectoryName(self);
             var selfName = Path.GetFileName(self);
             self = Path.GetFullPath(Path.Combine(selfDir, "../netstandard2.0/", selfName));
-            #endif
+#endif
             var refsPath = self + ".refs";
             var refs = File.ReadAllLines(refsPath).Concat(new[] {self});
             return new CecilTypeSystem(refs, null);
         }
 
         public CompilerTestBase() : this(CreateCecilTypeSystem())
         {
-            
+#if NETFRAMEWORK
+            // TODO: It's the hack for VS tests
+            var selfDirectory = Path.GetDirectoryName(typeof(CompilerTestBase).Assembly.Location);
```

Comment:
Can we check that we are running for Windows and using VS test platform? Or is it OK to do with `dotnet test` + Mono on Linux too?

Reply:
> Can we check that we are running for Windows and using VS test platform? 

I did not consider this, as it seemed that it would be easier to limit the .NET Framework. In addition, in my opinion, this does not really affect, except that VS for some reason changes `AppDomain.CurrentDomain.BaseDirectory`

> Or is it OK to do with dotnet test + Mono on Linux too?

I think not, besides https://github.com/mono/mono/issues/6984

But I am far from using mono

Reply:
Now I have done for everyone, but with verification `AppDomain.CurrentDomain.BaseDirectory`